WcaOnRails/lib/country_city_validators/gb_city_validator.rb

```diff
@@ -92,6 +92,17 @@
   Wiltshire
   Worcestershire
   Yorkshire
+  Akrotiri\ and\ Dhekelia
```

Comment:
I think the list is currently alphabetized, I think it would be good to continue to keep it alphabetized.

Reply:
The reason for not alphabetizing it was to keep the UK Counties and British Overseas Territories clearly separated. FWIW the British Overseas Territories are alphabetized in their own rite.

Reply:
Agree with Mollerz

Reply:
Ah, got it! If you wanted to rewrite this code to reflect that, you could do something like this:

```ruby
GB_COUNTIES = %w(
  ...
).to_set
GB_TERRITORIES = %w(
  ...
).to_set
GB_REGIONS = GB_COUNTIES | GB_TERRITORIES

CityValidator.add_validator_for_country "GB", CityCommaRegionValidator.new(type_of_region: "county", valid_regions: GB_REGIONS)
```

Not a big deal, though.

Reply:
@jfly's suggestion sounds good to me!

Reply:
Yes I like this! Can you guys proceed with this one?

Reply:
Sure!